### DISCUSSION TITLE
time to generate video

### DISCUSSION THREAD

**Discussion by eri0105** [OpenTalker/SadTalker#404] [General]
Hello everybody. I'm new to SadTalker and would like to know how long it takes to generate a 1 minute video. It has been generating a 1 minute video on my computer for hours. Am I doing something wrong?

**Comment by jsondn** [comment]
I am also facing same issue, let me know the correct process

**Reply by eri0105** [reply]
For me, I was only able to use SadTalker, after I changed my video card. Before I used a GTX 1050 TI, I changed it for an RTX 3060 12 gb. That's the only way I was able to use SadTalker. Cards less than 8gb do not support the app

**Reply by eri0105** [reply]
And even with a 12 gb video card, a 5 minute animation takes almost 1 hour

**Reply by mwreeb** [reply]
Using a Mac M1 (2020) 16GB RAM.  A minuted of video takes about an hour.

**Reply by tnfsmith** [reply]
Try to increase speed of render by install xformers==0.0.2
On Windows 11, 16GB RAM, GTX GeForce 1660Ti, I5-12500k, Its take about an hour 2 minutes

**Comment by U-DI-Page** [comment]
Maybe you are using cpu not gpu to run it . I just need few minutes

**Reply by Ruby141002** [reply]
how to change that setting to use cpu instead of gpu ?
